test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -166,3 +166,36 @@ Describe "Debug-job test" -tag "Feature" {
         $result.Command | Should be "<ScriptBlock>"
     }
 }
+
+Describe "Ampersand background test" -tag "CI" {
+    Context "Simple background job" {
+        AfterEach {
+            Get-Job | Remove-Job -force
+        }
+        It "Background with & produces a job object" {
+            $j = write-output Hi &
+            $j | Should BeOfType System.Management.Automation.Job
+        }
+        It "doesn't cause error when variable is missing" {
+            Remove-Item variable:name -erroraction Ignore
+            $j = write-output "Hi $name" &
+            rcjb $j -wait | Should Match "Hi "
+        }
+        It "Variables get copied" {
+            $name = "Bob"
+            $j = write-output "Hi $name" &
+            rcjb $j -wait | Should Match "Hi Bob"
+        }
+        It "doesn't overwrite the PID in the child process" {
+            $j = write-output $pid &
+            $cpid = rcjb $j -wait
+            $pid -ne $cpid | SHould Be $true
+        }
+        It "handles backgrounding expression" {
+            $j = 2+3 &
+            rcjb $j -wait | Should Be 5
```

Comment:
Are you sure the expression is evaluated only once and in the background.

From my quick review, I'm concerned it's evaluated twice, once in the foreground (and thrown away), and then again in the background.

Something like `$(Sleep -Seconds 10; 2+3) &` would make it obvious I think.